### ISSUE TITLE
Elevated circulating aspartate aminotransferase concentration

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9770]
**Preferred term label:**
Elevated circulating aspartate aminotransferase concentration

**Synonyms**
The reverse of those at HP:0031956

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/614419 (not really a disease; not sure why we covered this).

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal  -- Do you mean "Reduced circulating aspartate aminotransferase concentration"? Not sure I understand.

**Comment by MickeySegal** [comment]
Oops.  you are right.  I mean "Reduced circulating aspartate aminotransferase concentration"

**Comment by pnrobinson** [comment]
@MickeySegal 
added
HP:0430136
Reduced circulating aspartate aminotransferase concentration
